results.go

```diff
@@ -122,6 +124,34 @@ func (v *vocdoniHandler) finalizeElectionResults(election *api.Election, electio
 			return
 		}
 		log.Infow("final results image built ondemand", "electionID", election.ElectionID.String())
+		hubResults := &communityhub.HubResults{
+			Question:   electiondb.Question,
+			Options:    choices,
+			Date:       election.EndDate.String(),
+			CensusRoot: election.Census.CensusRoot,
+			CensusURI:  election.Census.CensusURL,
+		}
+		hubResults.Tally = make([][]*big.Int, 0)
+		for _, option := range election.Results[0] {
+			opt := new(big.Int).Set(option.MathBigInt())
+			hubResults.Tally[0] = append(hubResults.Tally[0], opt)
+		}
+		// hubResults.Turnout = big.NewInt(int64(electiondb.Turnout))
+		// hubResults.TotalVotingPower = big.NewInt(int64(election.Census.MaxCensusSize))
+		// hubResults.Participants =
```

Comment:
Why not add participants? What's the maximum size of participants we can add? Maybe we can check for a maximum and do it only if it's not bigger.

Reply:
there is no limit in size, but the smart contract needs to be changed as it is expecting fids and not usernames